src/pages/Autolending/AutolendingWhen.vue

```diff
@@ -176,7 +192,7 @@ export default {
 			if (donationToggle !== previousDonationToggle) {
 				if (donationToggle === 'on' && this.donation === 0) {
 					// set default donation if 0
-					this.donation = 15;
+					this.donation = this.hasAutoDepositDonation ? 0 : 15;
```

Comment:
Same comment as above. If the donation is getting zeroed out on the backend, then modifying these watches seems unnecessary since they won't even be shown.

Reply:
Reverted.